lib/main_common.pm

```diff
@@ -1811,6 +1806,8 @@ sub load_create_hdd_tests {
     loadtest 'console/integration_services' if is_hyperv;
     loadtest 'console/hostname'              unless is_bridged_networking;
     loadtest 'console/force_scheduled_tasks' unless is_jeos;
+    # setup $serialdev permission and so on
+    loadtest "console/consoletest_setup";
```

Comment:
Sorry but it does not make sense to schedule a test module called "consoletest_setup" that late after all the previous modules. I suggest you look into the proposal I provided to split the different parts of consoletest_setup into smaller chunks. Can you try that?

Reply:
Of course, but can you please merge fixed "consoletest_setup" first? In this way I will pull request another PR and close some #poo. But then I will continue #4909 and pull request a new one if finish.